supporting_content/dimensions/chargedescription.md

```diff
@@ -0,0 +1,33 @@
+# Column: ChargeDescription
+
+## Example provider mappings
+
+Current resource types found or extracted from available data sets:
+
+| Provider  | Data set                | Column                                            	|
+| :-------- | :---------------------- | :-------------------------------------------------------|
+| AWS       | CUR                     | lineItem/LineItemDescription 				|
+| GCP       | BigQuery Billing Export | sku.description                                         |
+| Microsoft | Cost details            | Not available      			                |
```

Comment:
I think `MeterName` is as close as it gets in Azure: [link](https://azure.microsoft.com/en-us/updates/update-reports-for-improved-meter-names/)

We can call it out as not a precise match because it requires other columns' values to have sufficient context, but this is what Ternary uses for this column.

Reply:
Actually, `ProductName` is the same thing as `sku.description`, but with a little more context. 
```suggestion
| Microsoft | Cost details            | ProductName      			                |
```

Reply:
Given that this dimension is meant to be a bit subjective, I don't think it makes sense to try to concoct a list of columns that tries to mimic AWS' or GCP's corresponding columns.  While MeterName and ProductName may contain important aspects of the entire line item, they likely won't encompass summary.  Thoughts?

Reply:
I think it is ok to call out that they all feel like a good fit sometimes, and there is not one exact fit in the current data format. That would be representative of my experience as a data platform creator.

Reply:
Just bumping here, I would be in favor of either setting this row to ProductName (accept Michael's suggestion), setting it to "ProductName and MeterName provide similar information", but i feel strongly it shouldn't be marked as 'unavailable'